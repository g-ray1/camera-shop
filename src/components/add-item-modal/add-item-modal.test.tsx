import { render } from '@testing-library/react';
import AddItemModal from './add-item-modal';

describe('AddItemModal component', () => {
  it('Snapshot', () => {
    const view = render(<AddItemModal />);

    expect(view).toMatchSnapshot();
  });
});
