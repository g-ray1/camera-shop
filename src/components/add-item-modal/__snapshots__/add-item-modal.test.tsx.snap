// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddItemModal component Snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <p
        class="title title--h4"
      >
        Добавить товар в корзину
      </p>
      <div
        class="basket-item basket-item--short"
      >
        <div
          class="basket-item__img"
        >
          <picture>
            <source
              srcset="/img/content/img9.webp, /img/content/img9@2x.webp 2x"
              type="image/webp"
            />
            <img
              alt="Фотоаппарат «Орлёнок»"
              height="120"
              src="/img/content/img9.jpg"
              srcset="/img/content/img9@2x.jpg 2x"
              width="140"
            />
          </picture>
        </div>
        <div
          class="basket-item__description"
        >
          <p
            class="basket-item__title"
          >
            Фотоаппарат «Орлёнок»
          </p>
          <ul
            class="basket-item__list"
          >
            <li
              class="basket-item__list-item"
            >
              <span
                class="basket-item__article"
              >
                Артикул:
              </span>
               
              <span
                class="basket-item__number"
              >
                O78DFGSD832
              </span>
            </li>
            <li
              class="basket-item__list-item"
            >
              Плёночная фотокамера
            </li>
            <li
              class="basket-item__list-item"
            >
              Любительский уровень
            </li>
          </ul>
          <p
            class="basket-item__price"
          >
            <span
              class="visually-hidden"
            >
              Цена:
            </span>
            18 970 ₽
          </p>
        </div>
      </div>
      <div
        class="modal__buttons"
      >
        <button
          class="btn btn--purple modal__btn modal__btn--fit-width"
          type="button"
        >
          <svg
            aria-hidden="true"
            height="16"
            width="24"
          >
            <use
              xlink:href="#icon-add-basket"
            />
          </svg>
          Добавить в корзину
        </button>
      </div>
    </div>
  </body>,
  "container": <div>
    <p
      class="title title--h4"
    >
      Добавить товар в корзину
    </p>
    <div
      class="basket-item basket-item--short"
    >
      <div
        class="basket-item__img"
      >
        <picture>
          <source
            srcset="/img/content/img9.webp, /img/content/img9@2x.webp 2x"
            type="image/webp"
          />
          <img
            alt="Фотоаппарат «Орлёнок»"
            height="120"
            src="/img/content/img9.jpg"
            srcset="/img/content/img9@2x.jpg 2x"
            width="140"
          />
        </picture>
      </div>
      <div
        class="basket-item__description"
      >
        <p
          class="basket-item__title"
        >
          Фотоаппарат «Орлёнок»
        </p>
        <ul
          class="basket-item__list"
        >
          <li
            class="basket-item__list-item"
          >
            <span
              class="basket-item__article"
            >
              Артикул:
            </span>
             
            <span
              class="basket-item__number"
            >
              O78DFGSD832
            </span>
          </li>
          <li
            class="basket-item__list-item"
          >
            Плёночная фотокамера
          </li>
          <li
            class="basket-item__list-item"
          >
            Любительский уровень
          </li>
        </ul>
        <p
          class="basket-item__price"
        >
          <span
            class="visually-hidden"
          >
            Цена:
          </span>
          18 970 ₽
        </p>
      </div>
    </div>
    <div
      class="modal__buttons"
    >
      <button
        class="btn btn--purple modal__btn modal__btn--fit-width"
        type="button"
      >
        <svg
          aria-hidden="true"
          height="16"
          width="24"
        >
          <use
            xlink:href="#icon-add-basket"
          />
        </svg>
        Добавить в корзину
      </button>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
